### PR TITLE
Fix Connections search issue

### PR DESCRIPTION
**Pull request by Yoshani**
1. Instead of giving the search results from the full authenticator list, all local authenticators that are not expected to be listed (using the list `localAuthenticatorList`) are filtered and removed.
2. A Beta label is added to the Active Sessions Limit authenticator for the SaaS deployment.

### Related Issues
- Issue https://github.com/wso2/product-is/issues/17695

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx [hunk 1]
```diff
@@ -390,6 +395,14 @@ export const Authenticators: FunctionComponent<AuthenticatorsPropsInterface> = (
...
                                 subHeader={ authenticator.categoryDisplayName }
                                 description={ authenticator.description }
+                                featureStatus={ isSAASDeployment && authenticator
+                                    .defaultAuthenticator?.authenticatorId === AuthenticatorManagementConstants
+                                    .ACTIVE_SESSION_LIMIT_HANDLER_AUTHENTICATOR_ID
+                                    ? (<Chip
+                                        size="small"
+                                        label={ t("common:beta").toUpperCase() }
+                                        className="oxygen-chip-beta"
+                                    />) : null }
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx:405]: Shall we move this to a seperate function, this is not that readable.

Comment (Yoshani) [reply apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx]: done

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx [hunk 2]
```diff
@@ -357,6 +362,31 @@ export const Authenticators: FunctionComponent<AuthenticatorsPropsInterface> = (
...
     };
 
+    /**
+     * Render feature status chip.
+     *
+     * @param authenticator - Authenticator.
+     *
+     * @returns Feature status chip.
+     */
+    const renderFeatureStatusChip = (authenticator: GenericAuthenticatorInterface): ReactElement => {
```
Comment (Achintha444) [inline apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx:372]: Can't we take the `authenticator ` directly, without passing it to the method ?

**Review by Achintha444 — APPROVED** [review]

File: apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx (hunk 2 above)
Comment (Yoshani) [reply apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx:372]: The authenticator is taken by iterating over the `authenticators` list https://github.com/wso2/identity-apps/blob/fc01f0e22bf4c7f6c344573961e097598d017d07/apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authenticators.tsx#L393. So we can't directly take it.

**Pull request merged**